staging/src/kubevirt.io/api/core/v1/schema.go

```diff
@@ -384,6 +387,15 @@ type Machine struct {
 	Type string `json:"type"`
 }
 
+type Backed struct {
+	// File backs the VM's memory by a file. Using this configuration allows the node to
```

Comment:
nit: `File backs the VM`s memory and a node cache is used to offset a performance hit by using slower storage . The node is able to reclaim the VM`s memory as it is just cache(disregarding dirty pages) and it is always stored on the disk. Note: For now the ephemeral storage is used to back this file.`